validity/src/env.rs

```diff
@@ -89,15 +89,13 @@ pub fn read_proposer_env() -> Result<EnvironmentConfig> {
     };
 
     // Parse proof mode
-    let agg_proof_mode = match get_env_var("AGG_PROOF_MODE", Some("groth16".to_string()))?
-        .to_lowercase()
-        .as_str()
-    {
-        "plonk" => SP1ProofMode::Plonk,
-        "groth16" => SP1ProofMode::Groth16,
-        "compressed" => SP1ProofMode::Compressed,
-        _ => SP1ProofMode::Groth16, // Default to Groth16 if no match
-    };
+    let agg_proof_mode =
```

Comment:
If I am not mistaken, the only change to this file is on format, not content. If so, does it make sense to add it to the PR?